### PR TITLE
readme: Add pointers to where to reach us

### PR DESCRIPTION
**Pull request by fabiand**
Signed-off-by: Fabian Deutsch <fabiand@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +66,16 @@ and learn more about the major components by taking a look at
  * [Components](docs/components.md) - Detailed look at all components
 
 
+# Community
+
+If oyu got enough of code and want to speak to people, then you got a couple
+of options:
+
+* Follow us on social media [Twitter](https://twitter.com/kubevirt)
```
Comment (rmohr) [inline README.md]: Follow us on [Twitter](https://twitter.com/kubevirt)

should be enough

File: README.md [hunk 2]
```diff
@@ -67,6 +66,16 @@ and learn more about the major components by taking a look at
  * [Components](docs/components.md) - Detailed look at all components
 
 
+# Community
+
+If oyu got enough of code and want to speak to people, then you got a couple
```
Comment (rmohr) [inline README.md]: s/oyu/you

File: README.md [hunk 3]
```diff
@@ -67,6 +66,16 @@ and learn more about the major components by taking a look at
  * [Components](docs/components.md) - Detailed look at all components
 
 
+# Community
```
Comment (rmohr) [inline README.md]: Maybe `Where to find us` ?

File: README.md [hunk 4]
```diff
@@ -67,6 +66,16 @@ and learn more about the major components by taking a look at
...
+
+If oyu got enough of code and want to speak to people, then you got a couple
+of options:
+
+* Follow us on social media [Twitter](https://twitter.com/kubevirt)
+* Ask questions and provide help on [IRC](irc://irc.freenode.net/#kubevirt)
```
Comment (rmohr) [inline README.md]: Maybe: `Chat with us on [IRC](irc://irc.freenode.net/#kubevirt)`

File: README.md [hunk 5]
```diff
@@ -67,6 +66,16 @@ and learn more about the major components by taking a look at
...
+* Find help and discuss on the [Google Group](https://groups.google.com/forum/#!forum/kubevirt-dev)
```
Comment (rmohr) [inline README.md]: Maybe `Discuss with us on the kubevirt-dev Google Group` ?

File: README.md (hunk 2 above)
Comment (fabiand) [inline README.md]: Thanks

File: README.md (hunk 1 above)
Comment (fabiand) [inline README.md]: Yep

File: README.md (hunk 4 above)
Comment (fabiand) [inline README.md]: Yep

File: README.md (hunk 5 above)
Comment (fabiand) [inline README.md]: Yeah ...

So - Previously I tried to give guidance where to put what stuff (ask questions, discuss, …) now it's just lisintg the channels. That#s fine for me tho - let's see how it goes.

File: README.md (hunk 3 above)
Comment (fabiand) [inline README.md]: Keeping it to community for now - I'll also mention the community repo and code of conduct (which I'll also add to the main repo)

**Comment by fabiand** [comment]
Thanks - Will update the PR.

**Pull request merged**